lib/CodecInfo.js

```diff
@@ -259,12 +263,12 @@ CodecInfo.expand = function(plain)
  * @param {Array<String>} names
  * @param {Boolean} rtx - Should we add rtx?
  * @param {Array<RTCPFeedbackInfoPlain>} rtcpfbs - RTCP feedback params
- * @returns {Map<String,CodecInfo>}
+ * @returns {Map<Number,CodecInfo>}
```

Comment:
I don't think this change is acually correct, we need to double check `answer()` behaviour